llvm/lib/CodeGen/LiveDebugVariables.cpp

```diff
@@ -715,6 +715,7 @@ static void printExtendedName(raw_ostream &OS, const DINode *Node,
     Res = V->getName();
     Line = V->getLine();
   } else if (const auto *L = dyn_cast<const DILabel>(Node)) {
+    // XXX what are we doing here? Adjust it?
```

Comment:
Has this note-to-self been addressed? It should be removed in any case.

Reply:
yes. I came to the conclusion that this is only debug output, and that printing column / coro-suspend-idx here would probably only hurt the readability of the debug output. As such, this code should stay unchanged.

Forgot the comment, though :/ Fixed now.